src/views/main.py

```diff
@@ -0,0 +1,188 @@
+from flask import render_template, request, redirect, url_for
+from src import app, mongo
+from src.datatable import DataTablesServer
+from flask_login import login_required, current_user
+from bson import ObjectId
+from functools import wraps
+import json
+
+
+# make connection with MongoDB data
+variant_col = mongo.db.variant
+user_col = mongo.db.user
+
+columns = ['#', '_id', 'total']
+default_fields = ["Details", "dn_no", "gene", "Position", "inheritanceMode", "cdna", "protein", "effect", "ref", 
+                  "genotype Patient", "genotype Mother", "genotype Father", "inheritedFrom", "GnomAD", "fullgnomen",
+                  "transcript"]
+default_order = {"dn_no": 1, "gene": 1}
+variants = []
```

Comment:
Are these variables required to be global variables?
And/or should they move to a `if __name__ == "__main__"` block

Reply:
it is used by multiple html files, basically all  the main pages (genes, patients, variants), so I think it is needed like this